### PR TITLE
ServiceNetworking Mgmt. SDK

### PR DESCRIPTION
**Pull request by malpaniHMC**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure ServiceNetworking management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/README.md:3]: Please add the description here

File: sdk/servicenetworking/ci.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
```
Comment (ArthurMa1978) [inline sdk/servicenetworking/ci.yml:3]: change the file name to `ci.mgmt.yml`

File: sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/CHANGELOG.md:3]: Change the `Unreleased` to the expect date when need us to release the lib to nuget (2 weeks after pr published at least)

File: sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/README.md (hunk 1 above)
Comment (malpaniHMC) [reply sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/README.md]: I've added a description for the time being, but I need to run this by our PM before we release this.

**Review by ArthurMa1978 — APPROVED** [review]

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**